countlyCommon/countlyCommon/CountlyBase.cs

```diff
@@ -1017,6 +1026,12 @@ public async Task ChangeDeviceId(String newDeviceId, bool serverSideMerge = fals
         internal bool consentRequired = false;
         internal Dictionary<ConsentFeatures, bool> givenConsent = new Dictionary<ConsentFeatures, bool>();
 
+        internal enum ConsentChangedAction
```

Comment:
Move this declaration (together with the decleration of the "ConsentFeatures" enum, "consentRequired ", "givenConsent ") to the top